apps/site/src/components/pages/hub/hub-utils.tsx

```diff
@@ -9,7 +9,8 @@ export type BlockSchema = Record<string, any>;
 
 /** @todo possibly extend this type */
 export type BlockExampleGraph = {
```

Comment:
This feels like it should be defined somewhere else, alongside an actual validation function, as otherwise we're just asserting that things exist all over the place and pages crash when they don't.